src/utils/helpers/semanticReactMarkdown.tsx

```diff
@@ -41,6 +41,12 @@ const MarkdownBlock: React.FC<MarkdownBlockProps> = (props) => {
         )
       },
     },
+    // Prevents returned HTML from being wrapped in <p></p> tags
+    noParagraph: {
+      paragraph: ({ node: { children } }: any) => {
+        return <span>{children[0].value}</span>
```

Comment:
I'm not sure about this one. It's guarantee this will always be receiving an array that has a value? It might be saffer to check that before and return null otherwise (with a log message for us).

Reply:
Okay, I've changed it to:
```
return <span>{children?.[0]?.value}</span>
```
So if any of the array or its element properties are missing, it'll just return an empty `<span></span>`

Same on the "heading" one above. 

c7a883a That okay?